test/snippet/alignment/scoring/aminoacid_scoring_scheme.cpp

```diff
@@ -0,0 +1,53 @@
+#include <range/v3/view/zip.hpp>
+
+#include <seqan3/alignment/scoring/aminoacid_scoring_scheme.hpp>
+#include <seqan3/alphabet/aminoacid/all.hpp>
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+
+{
+//! [two letters]
+aminoacid_scoring_scheme scheme{aminoacid_similarity_matrix::BLOSUM62};
+std::cout << "BLOSUM62 score for T and S: " << (int) scheme.get_score(aa27::T, aa27::S) << "\n"; // == 1
+
+scheme.set_similarity_matrix(aminoacid_similarity_matrix::BLOSUM80);
+std::cout << "BLOSUM80 score for aa27::T and aa20::S: " << (int) scheme.get_score(aa27::T, aa20::S) << "\n"; // == 2
+// you can score aa20 against aa27
+
+scheme.set_hamming_distance();
+std::cout << "Hamming distance between T and S: " << (int) scheme.get_score(aa27::T, aa20::S) << "\n"; // == -1
+std::cout << "Hamming distance between T and T: " << (int) scheme.get_score(aa27::T, aa20::T) << "\n"; // == 0
+//! [two letters]
+}
+
+{
+//! [edit matrix]
+aminoacid_scoring_scheme scheme;
```

Comment:
Why did you change from initializing the scheme directly with the matrix 
(`scheme.set_similarity_matrix(aminoacid_similarity_matrix::BLOSUM80);` [here](https://github.com/seqan/seqan3/pull/390/files#diff-6c0ebd7dd2a142616084565cf9b4315eL114) 
to default initialization and the `set_similarity_matrix()` function?

Reply:
It's just because the `edit matrix` example is in a different scope from the example above it, where `scheme` was declared. So I have to re-declare the `scheme` variable before I could do anything with it. I think `scheme.set_similarity_matrix(...)` in that context was used to change the similarity matrix from whatever it was before, I don't think it's initialized like that in the example.

Reply:
Yes all true, but still, in the original documentation block (https://github.com/seqan/seqan3/pull/390/files#diff-6c0ebd7dd2a142616084565cf9b4315eL114) it IS initilaized with the matrix not the function, so you might want to change it back.

Reply:
Done! :heavy_check_mark: